website/client-old/js/services/userServices.js

```diff
@@ -613,7 +633,7 @@ angular.module('habitrpg')
 
         authenticate: function (uuid, token, cb) {
           if (uuid && token) {
-            var offset = moment().zone(); // eg, 240 - this will be converted on server as -(offset/60)
+            var offset = moment().utcOffset(); // eg, 240 - this will be converted on server as -(offset/60)
```

Comment:
can you revert the changes to `utcOffset` back to `zone`? we already tried to change them but caused issues with out code so now we've a more broad open PR that should be merged soon that deals with all the problems